Mathlib/NumberTheory/NumberField/FinitePlaces.lean

```diff
@@ -111,28 +115,25 @@ def FinitePlace (K : Type*) [Field K] [NumberField K] :=
 noncomputable def FinitePlace.mk (v : HeightOneSpectrum (𝓞 K)) : FinitePlace K :=
   ⟨place (embedding v), ⟨v, rfl⟩⟩
 
-lemma toNNReal_Valued_eq_vadicAbv (x : K) :
-    toNNReal (norm_ne_zero v) (Valued.v (self:=v.adicValued) x) = vadicAbv v x := rfl
+lemma toNNReal_Valued_eq_vadicAbv (x : WithVal (v.valuation K)) :
+    toNNReal (norm_ne_zero v) (Valued.v x) = vadicAbv v x := rfl
 
 /-- The norm of the image after the embedding associated to `v` is equal to the `v`-adic absolute
 value. -/
-theorem FinitePlace.norm_def (x : K) : ‖embedding v x‖ = vadicAbv v x := by
-  simp only [adicCompletion, NormedField.toNorm, instNormedFieldValuedAdicCompletion,
-    Valued.toNormedField, instFieldAdicCompletion, Valued.norm, Valuation.RankOne.hom,
-    MonoidWithZeroHom.coe_mk, ZeroHom.coe_mk, embedding, UniformSpace.Completion.coeRingHom,
-    RingHom.coe_mk, MonoidHom.coe_mk, OneHom.coe_mk, Valued.valuedCompletion_apply,
-    toNNReal_Valued_eq_vadicAbv]
+theorem FinitePlace.norm_def (x : WithVal (v.valuation K)) : ‖embedding v x‖ = vadicAbv v x := by
```

Comment:
I think that this result about the norm, and its companions, should appear in the doc at the top of the file.